packages/terra-framework-docs/src/terra-dev-site/doc/tabs/UpgradeGuide.2.doc.mdx

```diff
@@ -4,6 +4,12 @@ import { Badge } from 'terra-tabs/package.json?dev-site-package';
 
 # Terra Tabs Upgrade Guide
 
+## Changes from version 6 to version 7
+
+### Removed props
+
+- Removed responsiveTo
```

Comment:
this guide need to include few details of consumption guide of common tabs along with new `variant` prop to switch between framework and workspace styles.

Reply:
On the consumption side, terra-tabs users would continue to use as before, there won't be any changes needed to be applied while using framework API. And the variant prop won't be exposed to user, since it is internally used to switch the styles